geniza/corpus/locale/ar/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,40 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
Do we need to configure something so that these will be populated with meaningful values? Or are we expected to edit and change them manually?

Reply:
I think you have to edit them manually, unfortunately.

Reply:
OK, not quite – you have to [implement a custom management command as detailed here](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#customizing-makemessages) so that you can control the [parameters that are passed to GNU `xgettext`](https://www.gnu.org/software/gettext/manual/html_node/xgettext-Invocation.html). Which might be worthwhile so that we can also ignore files we don't want to translate, which is kind of a problem on the command-line.

Reply:
I searched through [django's settings](https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-USE_I18N) options and couldn't find anything. It also looks like Apple's open source asks their translators to manually enter this information! So maybe this isn't a feature that's been implemented yet? https://opensource.apple.com/source/subversion/subversion-52/subversion/TRANSLATING.auto.html

Should we just ignore this for now?

Reply:
Ignore me! I hadn't refreshed the page when @thatbudakguy had made his comment